packages/hasher/src/index.ts

```diff
@@ -107,6 +104,18 @@ export class Hasher implements IHasher {
   }
 
   public async hashOfOutput(): Promise<string> {
-    return generateHashOfFiles(this.packageRoot, this.outputGlob);
+    // ensure this.repoInfo is generated
+    if (!this.repoInfo) {
+      throw new Error("make sure the createPackageHash is called first");
+    }
+
+    return generateHashOfFiles(
```

Comment:
Hmmm, this is troublesome. This is called when users pass in "validateOutput" option. I can make this call a "getRepoInfoWithNoCache." Since this option isn't a typical option to be used in production, I'll just this to a non-caching getRepo.

Reply:
This`validateOutput` option is not used anymore by anyone and has been annoying me as well in the past. Maybe we should cut the cost and remove this unused feature.